docs/index.rst

```diff
@@ -110,7 +110,7 @@ https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
 
 The Python bindings also provide some additional configuration properties:
 
-* ``default.topic.config``: value is a dict of client topic-level configuration
+* **DEPRECATED** ``default.topic.config``: value is a dict of client topic-level configuration
```

Comment:
Needs to mention that topic configuration now goes on the global config dictionary.